package.json

```diff
@@ -1,6 +1,17 @@
 {
-  "private": true,
   "name": "tupaia",
+  "private": true,
+  "description": "Mono-repo for the Tupaia project, mapping health data across the Pacific",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/beyondessential/tupaia.git"
+  },
+  "author": "Beyond Essential Systems <admin@tupaia.org> (https://beyondessential.com.au)",
+  "license": "SEE LICENSE IN LICENSE",
+  "bugs": {
+    "url": "https://github.com/beyondessential/tupaia-backlog/issues"
```

Comment:
We should create a public issue system - it could be github or even some other support platform, and we can triage and process bugs from there into our github issue backlog. [I just found your note on the issue and added an additional note]

Reply:
I agree, I think that @kujo4pm has some ideas around that